includes/documents/class-wcpdf-invoice.php

```diff
@@ -411,6 +413,37 @@ public function init_settings() {
 					'description'	=> sprintf(__( "Disable document when the order total is %s", 'woocommerce-pdf-invoices-packing-slips' ), function_exists('wc_price') ? wc_price( 0 ) : 0 ),
 				)
 			),
+			array(
+				'type'			=> 'setting',
+				'id'			=> 'mark_printed',
+				'title'			=> __( 'Mark printed', 'woocommerce-pdf-invoices-packing-slips' ),
+				'callback'		=> 'select',
+				'section'		=> 'invoice',
+				'args'			=> array(
+					'option_name' => $option_name,
+					'id'          => 'mark_printed',
+					'options'     => array_merge(
```

Comment:
I would add a description to this setting too, e.g. _"If enabled, allows you to mark the document as printed, manually (in the order page) or automatically (based on the document creation context you have selected)."_

@dwalkerpriv @dpeyou, what do you think?

Reply:
@YordanSoares yes, this would be good and the description you provided here is well also 👍